release-1.1.2-schemas-fixup/oscal_assessment-plan_schema.json

```diff
@@ -0,0 +1,4063 @@
+{
```

Comment:
I'm not sure I understand why there is a need to an explicit copy of the schema in a 1.1.2 project directory.


Aren't these relied on from the submodules

Reply:
@butler54 so, while we were doing the transition we figured out that instead of pointing directly to the project directory submodule, NIST is now including the schemas as part of the artifacts generated from the release, let me share documentation here: https://github.com/usnistgov/OSCAL/blob/main/build/README.md#artifact-generation